site/app/libraries/database/DatabaseQueries.php

```diff
@@ -7275,8 +7275,20 @@ public function getCurrentQueue() {
         $this->course_db->query($query, [$this->core->getDateTimeNow()->format('Y-m-d')]);
         return $this->course_db->rows();
     }
-
     public function getPastQueue() {
+        $off  = "ORDER BY ROW_NUMBER";
+        $up   = "ORDER BY helper.user_id";
+        $down = "ORDER BY helper.user_id DESC";
+        $orderType = "";
+        if ($_COOKIE['sortStateTimeEntered'] == "up") {
```

Comment:
I'm getting a warning about 'sortStateTimeEntered' being an undefined array key when I access the queue page. I second William's question about sorting on the client side and not using cookies.